poetry_release/plugin.py

```diff
@@ -1,16 +1,15 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
-
 from poetry.plugins.application_plugin import ApplicationPlugin
 
 from poetry_release.command import ReleaseCommand
 
-if TYPE_CHECKING:
-    from poetry.console.commands.command import Command
+
+def factory():
+    return ReleaseCommand()
```

Comment:
Use the latest recommended way of building command from `poetry`'s documentation